javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -355,45 +354,53 @@ public static ExecutableElement elementFromUse(NewClassTree node) {
   }
 
   /**
-   * Determines the symbol for a constructor given an invocation via {@code new}.
+   * Returns the super constructor invoked by {@code newClassTree}.
```

Comment:
I think this returns the constructor directly invoked, unless an anonymous class is being constructed, in which case the super constructor is returned.  Is that accurate?